src/index.html

```diff
@@ -9,6 +9,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="site-header">
+      <a href="/" class="brand">
+        <img
+          src="images/logo.png"
+          alt="brand"
+          class="logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Смартфоны и телефоны</a>
+          </li>
+          <li class="nav__item">
+            <a
+            href="#"
+            class="nav__link"
+            data-qa="hover"
+          >
+          Ноутбуки и компьютеры
+            </a>
```

Comment:
```suggestion
            <a
            href="#"
            class="nav__link"
            data-qa="hover"
            >
              Ноутбуки и компьютеры
            </a>
```
Неправильні відступи